packages/flutter/lib/src/widgets/scroll_controller.dart

```diff
@@ -82,7 +88,11 @@ class ScrollController {
   ///
   /// Immediately after the jump, a ballistic activity is started, in case the
   /// value was out of range.
-  void jumpTo(double value) => position.jumpTo(value);
+  void jumpTo(double value) {
+    assert(_positions.isNotEmpty, 'ScrollController not attached to any scroll views.');
+    for (ScrollPosition p in _positions)
```

Comment:
`s/p/position/`  (please use complete words in variable names)

Also, `jumpTo` generates scroll notifications synchronously, which could run arbitrary code that mutates `_positions` during this loop.  To avoid any shenanigans, I'd recommend using `new List<ScrollPosition>.from(_positions)` to make a copy of the list for the iteration.